deploy/olm-catalog/prod/7.13.5-1/manifests/businessautomation-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,423 @@
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          - routes/custom-host
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - build.openshift.io
+          resources:
+          - buildconfigs
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - image.openshift.io
+          resources:
+          - images
+          - imagestreams
+          - imagestreamimages
+          - imagestreamtags
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - app.kiegroup.org
+          resources:
+          - kieapps
+          - kieapps/status
+          - kieapps/finalizers
+          verbs:
+          - create
+          - delete
+          - deletecollection
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+        - apiGroups:
+          - operators.coreos.com
+          resources:
+          - clusterserviceversions
+          - subscriptions
+          verbs:
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: business-automation-operator
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - kieapp
+  - pam
+  - decision
+  - kie
+  - cloud
+  - bpm
+  - process
+  - automation
+  - operator
+  labels:
+    alm-owner-businessautomation: businessautomation-operator
+    operated-by: businessautomation-operator.7.13.5-1
+  links:
+  - name: Product Page
+    url: https://access.redhat.com/products/red-hat-process-automation-manager
+  - name: Documentation
+    url: https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/#category-deploying-red-hat-process-automation-manager-on-openshift
+  maintainers:
+  - email: bsig-cloud@redhat.com
+    name: Red Hat
+  maturity: stable
+  provider:
+    name: Red Hat
+  replaces: businessautomation-operator.7.13.4-5
```

Comment:
The replaces should be: `businessautomation-operator.7.13.4-1`

I suppose you forgot to update the file `version/version.go` so the changes would be properly propagated here.

Reply:
indeed, will update it later.

Reply:
updated.